addons/sale/models/sale_order_line.py

```diff
@@ -1044,7 +1044,8 @@ def _check_line_unlink(self):
     @api.ondelete(at_uninstall=False)
     def _unlink_except_confirmed(self):
         if self._check_line_unlink():
-            raise UserError(_("You can not remove an order line once the sales order is confirmed.\nYou should rather set the quantity to 0."))
+            raise UserError(_("Once a sales order is confirmed, you can not remove one of its line (we need to track if something gets invoiced or delivered).\
```

Comment:
```suggestion
            raise UserError(_("Once a sales order is confirmed, you cannot remove one of its line (we need to track if something gets invoiced or delivered).\
```
While both are correct, cannot is far more common